src/patterns/data/default/artery_and_arteriole_pattern.tsv

```diff
@@ -18,4 +18,6 @@ UBERON:8600098	dorsal lingual artery	FMA:49546	UBERON:0001637	UBERON:0001456	"An
 UBERON:8600099	accessory meningeal artery	FMA:79467	UBERON:0001637	UBERON:0000033	"An artery that branches from the maxillary artery and supplies the meninges. The accessory meningeal artery ascends through the foramen ovale to enter the cranial cavity and supply the dura mater of the floor of the middle cranial fossa and of the trigeminal cave, and to the trigeminal ganglion."	wikipedia:Accessory_meningeal_artery			UBERON:0001616	UBERON:0010743	https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
 UBERON:8600100	artery of pterygoid canal	FMA:85528	UBERON:0001637	UBERON:0001456	"An artery that branches from the maxillary artery and supplies blood to structures in the auditory tube, tympanic cavity, and upper pharynx. The artery of the pterygoid canal courses through the pterygoid canal, a small bony canal located in the sphenoid bone of the skull."	wikipedia:Artery_of_the_pterygoid_canal			UBERON:0001616	UBERON:0006562|UBERON:0001756|UBERON:0002393	https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
 UBERON:8600101	descending palatine artery	FMA:49791	UBERON:0001637	UBERON:0001456	An artery that branches from the maxillary artery and supplies the hard and soft palate.	wikipedia:Descending_palatine_artery			UBERON:0001616	UBERON:0003216|UBERON:0001733	https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
-UBERON:8600102	left anterior descending artery	FMA:3862	UBERON:0001637	UBERON:0000915	"An artery that branches from the left coronary artery and supplies the anterior wall of the left ventricle, the interventricular septum, and the apex of the heart."	wikipedia:Left_anterior_descending_artery	anterior interventricular branch of left coronary artery	FMA:3862	UBERON:0001626		https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
\ No newline at end of file
+UBERON:8600102	left anterior descending artery	FMA:3862	UBERON:0001637	UBERON:0000915	"An artery that branches from the left coronary artery and supplies the anterior wall of the left ventricle, the interventricular septum, and the apex of the heart."	wikipedia:Left_anterior_descending_artery	anterior interventricular branch of left coronary artery	FMA:3862	UBERON:0001626		https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
+UBERON:8600106	anterior humeral circumflex artery	FMA:22680	UBERON:0001637	UBERON:0001467	"An artery that branches from the axilliary artery and supplies the deltoid, biceps brachii, and coracobrachialis muscles, the glenohumeral joint, the proximal humeral head, and the skin overlying the anterior aspect of the proximal humerus. This artery courses laterally across the anterior surface of the humerus and anastomoses with the posterior humeral circumflex artery."	https://www.elsevier.com/resources/anatomy/cardiovascular-system/arteries/anterior-circumflex-humeral-artery/20091			UBERON:0001394	UBERON:0001476|UBERON:0001507|UBERON:0001505|UBERON:0001470|UBERON:0006801	https://orcid.org/0000-0001-6677-8489	2024-08-20T09:53:48Z

```

Comment:
You say in the definition: "supplies ... the skin overlying the anterior aspect of the proximal humerus".
Should we add the [upper arm skin](http://purl.obolibrary.org/obo/UBERON_0004263) to the `vessel_supplies_blood_to` list?

You might want to add `arteria circumflexa anterior humeri` (`synonym_xrefs`: `https://ta2viewer.openanatomy.org/?id=4630`).

Reply:
> You say in the definition: "supplies ... the skin overlying the anterior aspect of the proximal humerus".
Should we add the [upper arm skin](http://purl.obolibrary.org/obo/UBERON_0004263) to the vessel_supplies_blood_to list?

Thank you, I will add that term to be 'supplied'.

> You might want to add arteria circumflexa anterior humeri (synonym_xrefs: https://ta2viewer.openanatomy.org/?id=4630).

I was thinking about it, but the reason for not adding it was that it should have an annotation property 'has synonym type' latin, and I thought that would overcomplicate the DOSDP.